apps/nextjs-website/src/components/PressRelease.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import MarkdownRenderer from './MarkdownRenderer';
+import { PressReleaseSection } from '@/lib/fetch/types/PageSection';
+import { PressRelease as PressReleaseRC } from '@react-components/components';
+import { PressReleaseProps } from '@react-components/types';
+
+const makePressReleaseProps = ({
+  eyelet,
+  title,
+  subtitle,
+  body,
+  ctaText,
+  ctaHref,
+  sectionID,
+}: PressReleaseSection): PressReleaseProps => ({
+  title,
+  eyelet: eyelet || '',
+  ...(subtitle && { subtitle: MarkdownRenderer({ markdown: subtitle }) }),
+  body: MarkdownRenderer({ markdown: body }),
+  ctaText: ctaText || '',
+  ctaHref: ctaHref || '',
+  sectionID: sectionID || '',
+});
```

Comment:
```suggestion
const makePressReleaseProps = ({
  eyelet,
  subtitle,
  body,
  ...rest,
}: PressReleaseSection): PressReleaseProps => ({
  ...( eyelet && { eyelet },
  ...(subtitle && { subtitle: MarkdownRenderer({ markdown: subtitle }) }),
  body: MarkdownRenderer({ markdown: body }),
  ...rest,
});
```
Condense all fields that don't get updated into `...rest`